runner/runners/README.md

```diff
@@ -0,0 +1,29 @@
+# Runner Impl Design #
+This package contains implementations of Runner.
```

Comment:
Add a sentence about what Runner does.  If you don't know this doc provides no context